### ISSUE TITLE
Explicitly marking arguments as "okay to be unused"

### ISSUE THREAD

**Issue by bear24rw** [nhoad/flake8-unused-arguments#5]
I have some function where it is okay that I am not using the argument. It would be nice to easily mark those arguments so they don't get flaged. 

Possible solutions:

1)  if argument name starts with `_` then ignore it
```python
def foo(_bar):
    print("_bar is ignored")
```

2) `Unused` type annotation
```python
from something import Unused
def foo(bar: Unused):
    print("bar is ignored")
```

**Comment by bear24rw** [comment]
I just realized option 1 is already supported if I disable `U101`, closing!
